spek-data-driven-extension/src/test/kotlin/org/jetbrains/spek/data_driven/DataDrivenSpec.kt

```diff
@@ -0,0 +1,45 @@
+package org.jetbrains.spek.data_driven
+
+import org.jetbrains.spek.api.Spek
+import org.jetbrains.spek.api.dsl.given
+import org.jetbrains.spek.api.dsl.it
+import kotlin.test.assertEquals
+
+class DataDrivenSpec : Spek({
```

Comment:
I think we need to add tests for DataDriven thing to check that it actually executes tests, kinda like what @raniejade does in such PRs https://github.com/JetBrains/spek/pull/184